iperf.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, tc: TestConfig, ts: TestSettings):
 
         if self.exec_persistent:
             self.template_args["command"] = IPERF_EXE
-            self.template_args["args"] = ["-s", "-p", f"{self.port}"]
+            self.template_args["args"] = ["-s", "-p", f"{self.port}"]  # type: ignore
```

Comment:
this seems wrong to me.

Why is this done?

Maybe the type should be `dict[str, str|list[str]]` or `Mapping[str, str | list[str]]`.

Reply:
I tried a `dict[str, str|list[str]` at one point, but it made many more problems than it solved. 
Since args is the only thing that's a `list[str]`, I thought I defined at one point to use `self.args` instead of `self.template_args["args"]`. Must have gotten lost.